modules/nf-core/merqury/merqury/main.nf

```diff
@@ -56,4 +57,30 @@ process MERQURY {
         merqury: $VERSION
     END_VERSIONS
     """
+
+    stub:
+    prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = 1.3
```

Comment:
As simon said, please add the version blurb/comment here too

Reply:
That's done now.